tests/conftest.py

```diff
@@ -555,6 +555,46 @@ def mocked_setup_sub_logger(mocker):
         'bigchaindb.log.setup.setup_sub_logger', autospec=True, spec_set=True)
 
 
+@pytest.fixture
+def abci_http(tendermint_host, tendermint_port):
```

Comment:
I see this fixture is used in combination with `abci_server`, the question is: does it make sense to use `abci_http` alone? If not, I'd unify it with `abci_server`.